recipes/_omnibus_toolchain.rb

```diff
@@ -56,5 +56,6 @@
   end
 end
 
+certs_path = ::File.join(toolchain_install_dir, 'embedded', 'ssl', 'certs', 'cacert.pem')
```

Comment:
This extra `File.join` is redundant as it is executed as the first line of the `windows_safe_path_join` method.